rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -246,7 +246,7 @@ def output_specs_exploration(self) -> SpecDict:
         specs = {SampleBatch.ACTION_DIST: self.__get_action_dist_type()}
         if self._is_discrete:
             specs[SampleBatch.ACTION_DIST_INPUTS] = {
-                "logits": TorchTensorSpec("b, h", h=self.config.action_space.n)
+                "logits": TorchTensorSpec("b, h", h=int(self.config.action_space.n))
```

Comment:
curious to know why `.n` is not an int already?

Reply:
its a numpy.int64